src/applications/coronavirus-vaccination/config/uiSchema.js

```diff
@@ -160,8 +172,9 @@ export default {
       'ui:description': () => (
         <span>
           <b>Note: </b>
-          Your SSN helps us match your information to your Veteran records so we
-          can better understand your needs.
+          Your SSN helps us match your information to your Veteran records. We
```

Comment:
Same SSN, but we don't need the `title` this time because it's defined earlier in the page

Reply:
Only one of these elements will render (based on authenticated or not) so I included the title in both.  Thanks!